content-repo/extra-docs/packs/content-management.md

```diff
@@ -244,18 +806,23 @@ Currently, the pack does not support the following features:
 2. Server configuration keys.
 
 
-### Demo Video
-<video controls>
-    <source src="https://github.com/demisto/content-assets/raw/c332ede923f40990749a1498c6501fcacbf58bfd/Assets/ContentManagement/content_management_demo.mp4"
-            type="video/mp4"/>
-    Sorry, your browser doesn't support embedded videos. You can download the video at: https://github.com/demisto/content-assets/blob/master/Assets/ContentManagement/content_management_demo.mp4 
-</video>
 
+### CI/CD FAQs
+
+1. **We use Dev/Prod.  Do I still need this feature?  What is the difference?**
+
+    This feature is an alternative for the Dev/Prod environment. Instead of building and maintaining your code on a development Cortex XSOAR platform, you can do so from your own chosen git repository and utilize 3rd party tools like CI/CD infrastructures, build servers, artifact servers and more. This adds to your development and testing process. These multiple powerful tools do not exist with the Cortex XSOAR UI feature like Version control, code review, distributed developing environments, automatic testing, etc.
```

Comment:
```suggestion
    This feature is an alternative for the Dev/Prod environment. Instead of building and maintaining your code on a development Cortex XSOAR platform, you can do so from your own chosen Git repository and utilize 3rd party tools like CI/CD infrastructures, build servers, artifact servers and more. This adds to your development and testing process. These multiple powerful tools do not exist with the Cortex XSOAR UI feature like version control, code review, distributed developing environments, automatic testing, etc.
```